runtests.py

```diff
@@ -54,27 +55,21 @@ def is_class(string):
 
 
 if __name__ == "__main__":
-    try:
+    run_flake8 = True
+    run_isort = True
+    with suppress(ValueError):
         sys.argv.remove('--nolint')
-    except ValueError:
-        run_flake8 = True
-        run_isort = True
-    else:
         run_flake8 = False
         run_isort = False
 
-    try:
+    run_tests = True
+    with suppress(ValueError):
         sys.argv.remove('--lintonly')
-    except ValueError:
-        run_tests = True
-    else:
         run_tests = False
 
-    try:
+    style = 'default'
```

Comment:
what this `default` means?

Reply:
@drsantos20 this `style` is used to determine which way pytest will be run.

```python
PYTEST_ARGS = {
    'default': [],
    'fast': ['-q'],
}
```